SVA/lib/mmu.c

```diff
@@ -1392,6 +1392,7 @@ void
 sva_mm_load_pgtable (void * pg_ptr) {
   /* Cast the page table pointer to an integer */
   uintptr_t pg = (uintptr_t) pg_ptr;
+  u_long data = 0;
```

Comment:
This should be a uintptr_t.  This will ensure that the register is the same size as the system's pointer type.